.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: php
+services:
+  - mysql
+
+before_install:
+  - ./tests/ci-scripts/prepare-db.sh
+  - ./tests/ci-scripts/install-ojs.sh
```

Comment:
I'd suggest aligning this more with e.g. https://github.com/pkp/oaiJats/blob/master/.travis.yml -- that script can handle different branch names, DBMSs, etc., and makes use of the scripts built into the pkp-lib repo for much of the setup.